src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -981,16 +981,24 @@ List<Exception> generalResolutionExceptions
 
                         if (conflictCandidate.IsConflictVictim)
                         {
-                            LogConflict(conflictCandidate, fusionName);
+                            MessageImportance importance = ChooseReferenceLoggingImportance(conflictCandidate);
+                            LogConflict(conflictCandidate, fusionName, (string s, object[] parameters) =>
+                            {
+                                string code = Log.ExtractMessageCode(Log.FormatResourceString(s, parameters), out string message);
+                                if (code == null)
+                                    Log.LogMessageFromResources(importance, s, parameters);
+                                else
+                                    Log.LogMessage(importance, message);
+                            });
 
                             // Log the assemblies and primary source items which are related to the conflict which was just logged.
                             Reference victor = dependencyTable.GetReference(conflictCandidate.ConflictVictorName);
 
                             // Log the winner of the conflict resolution, the source items and dependencies which caused it
-                            LogReferenceDependenciesAndSourceItems(conflictCandidate.ConflictVictorName.FullName, victor);
+                            LogReferenceDependenciesAndSourceItems(conflictCandidate.ConflictVictorName.FullName, victor, (string s, object[] parameters) => Log.LogMessageFromResources(MessageImportance.Low, s, parameters));
```

Comment:
I don't think so, unless I'm ok with using more memory in the normal case, which you said I shouldn't. I'm not too worried about other messages getting interleaved just because the warning (if a warning comes) would be coherent, and I imagine almost everyone would look at that and ignore the rest of the logs, since it now has more details.